### ISSUE TITLE
Can't configure matrix account

### ISSUE THREAD

**Issue by gcampax** [Stanford-Mobisocial-IoT-Lab/thingengine-core#18]
_From @szimszon on July 15, 2018 20:18_

Hi,

I wonder if you could help me.

I tried to set up a matrix account in Almond with "configure matrix" it asks me for e-mail or phone number. I earlier noticed that Almond recognise matrix account like @localpart:domain.org so I tried with a custom made account like @almond:oregpreshaz.eu (the actual HS is mtrx.oregpreshaz.eu and is defined in DNS SRV record) after I'm asked about the password I typed and hit enter but nothing happens. Even I can't see anything in my server's log... Do this assistant work only with HS: matrix.org? Do I have to set e-mail address for my almond matrix account?

_Copied from original issue: Stanford-Mobisocial-IoT-Lab/thingengine-platform-android#70_

**Comment by gcampax** [comment]
That is correct, the current code is only set to work with the matrix.org home server.
I am going to transfer this issue to the underlying library containing the matrix communication code.

We also require people to sign up with a 3pid (either email or phone) because it allows us to tie the matrix identity to the Android address book, so that we can show a trusted name rather than a username for incoming requests.

Note though that Matrix is only useful for "Communicating Assistant" functionality, which allows you to share your third-party devices and services with other users that also use Almond (using hidden messages). You cannot, at this time, use Almond to send and receive messages directly on Matrix.

**Comment by szimszon** [comment]
Thanks for considering to extend the matrix messaging functionality.
